invenio_search_ui/assets/js/invenio_react_search_ui/Results.js

```diff
@@ -0,0 +1,146 @@
+// /*
+//  * This file is part of Invenio.
+//  * Copyright (C) 2020 CERN.
+//  *
+//  * Invenio is free software; you can redistribute it and/or modify it
+//  * under the terms of the MIT License; see LICENSE file for more details.
+//  */
+
+import React, { Component } from "react";
+import { Grid, Card, Image, Item, Label, Divider } from "semantic-ui-react";
+import _truncate from "lodash/truncate";
+import {
+  ActiveFilters,
+  Count,
+  LayoutSwitcher,
+  Pagination,
+  ResultsMultiLayout,
+  ResultsList,
+  ResultsGrid,
+  ResultsPerPage,
+  Sort
+} from "react-searchkit";
+
+const SpanWithMargin = ({ text, margin }) => {
```

Comment:
This can all be removed now with the support of the `label` prop to most of the components.